src/pages/RFOX/components/AddressSelection.tsx

```diff
@@ -116,7 +139,8 @@ export const AddressSelection: FC<AddressSelectionProps> = ({
     return (
       <Input
         {...register('manualRuneAddress', {
-          minLength: 1,
+          // Only required if user doesn't have a rewards address yet
+          required: !currentRuneAddress,
```

Comment:
More of a paranoia than anything else.
We will not render the manual field input if `currentRuneAddress` (disabled account selection will be rendered instead, or a tag for EVM wallets), so we should be safe with `required: true` but better be safe than sorry by being explicit.